providers/azure/resources/cloud_defender.go

```diff
@@ -454,19 +454,21 @@ func (a *mqlAzureSubscriptionCloudDefenderService) securityContacts() ([]interfa
 	if err != nil {
 		return nil, err
 	}
+	res := []interface{}{}
 	list, err := getSecurityContacts(ctx, armConn)
 	if err != nil {
-		return nil, err
+		// https: //github.com/mondoohq/cnquery/issues/4997
+		log.Warn().Err(err).Msg("fail gracefully")
```

Comment:
Why is that a warn?

Reply:
I was debating between this an a simple debug log message, I fee that is a better option, would you agree?